node/cmd/ccq/query_server.go

```diff
@@ -89,7 +87,16 @@ var QueryServerCmd = &cobra.Command{
 
 func runQueryServer(cmd *cobra.Command, args []string) {
 	if *verifyPermissions {
-		_, err := parseConfigFile(*permFile, *allowAnything)
+		env, err := common.ParseEnvironment(*envStr)
```

Comment:
should the env parsing just be moved above this (and not user the logger, as you've done here) to avoid the code duplication?